app/boards/shields/reviung5/reviung5.overlay

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+    chosen {
+            zmk,kscan = &kscan0;
+            zmk,matrix_transform = &default_transform;
+    };
+
+    default_transform: keymap_transform_0 {
+        compatible = "zmk,matrix-transform";
+        columns = <5>;
+        rows = <1>;
+
+        map = <RC(0,0) RC(0,1) RC(0,2) RC(0,3) RC(0,4)>;
+    };
+
+    kscan0: kscan_0 {
+            compatible = "zmk,kscan-gpio-matrix";
```

Comment:
I'm flabbergasted... It doesn't just use direct pins?

Reply:
I'm not sure - are you asking if it really uses a matrix? Then yes. One row with five columns.